inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/convert_like.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/convert_like.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+std::vector<std::vector<std::vector<size_t>>> inShapes = {
```

Comment:
Sam comments as for Convert case, but here you produce twice more permutations of shapes for both parameters which in fact will not affect operation behaviour.
Precisions combinations much more important for the operation

Reply:
Agree, TBD

Reply:
Done